python/tests/test_balance_metrics.py

```diff
@@ -88,6 +109,19 @@ def test_b1(self, ts):
         for tree in ts.trees():
             assert tree.b1_index() == pytest.approx(b1_index_definition(tree))
 
+    @pytest.mark.parametrize("ts", get_example_tree_sequences())
```

Comment:
Let's add another pytest.parametrize here like so:

```
@pytest.mark.parametrize("ts", get_example_tree_sequences())
@pytest.mark.parametrize("base", [2, 10, math.e, np.array([3])[0]])
def test_b2_base(self, ts, base):
    # as above

Reply:
Thanks for the input @jeromekelleher! I did the changes.